src/Sylius/Bundle/AdminBundle/Resources/views/Product/Tab/_details.html.twig

```diff
@@ -29,13 +29,39 @@
             </div>
             <div class="column">
                 {{ form_row(form.channels) }}
-
-                {% if product.simple %}
-                    <h4 class="ui dividing header">{{ 'sylius.ui.pricing'|trans }}</h4>
-                    {{ form_row(form.variant.channelPricings, {'label': false}) }}
-                {% endif %}
             </div>
         </div>
+        {% if product.simple %}
```

Comment:
Should be duplicated on variant page as well. Or we can try to extract common logic from this file and add it to variants as well.

Reply:
The variant page doesn't have a form though.

Reply:
Going to do that as well.

Reply:
I extracted it as a template. But I am not sure where to put it. I just for now put it in the Product folder, but I am open for suggestions.